src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -83,28 +123,53 @@ private FieldBuilder(EventStreamFieldType fieldType)
             return this;
         }
 
+        /// <summary>
+        /// Sets the resolver for the field.
```

Comment:
I suggest to throw `InvalidOperationException` if `FieldType` is not an input type.

Reply:
This appears to be a duplicate of the above comment, not intended for this member.  If you have another comment that applies here, please let me know.